selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/ConnectionsManagement.java

```diff
@@ -69,8 +75,8 @@ public void connectToAUser(String username, String... name) {
       searchPeople(name[0], null, null, null);
     else
       searchPeople(username, null, null, null);
-    evt.clickByJavascript(ELEMENT_CONNECTION_CONNECT_BTN.replace("${user}", username));
-    evt.waitForAndGetElement(ELEMENT_CONNECTION_CANCEL_BTN.replace("${user}", username), 2000, 1);
+$(byText("Connect")).click();
```

Comment:
Please create a new variable in SocialLocator instead of using string, to be compliant with other tests.